PeerioTechnologies/peerio-mobile
# fix: remove user from volume animation [branch ch8625]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/8625/mobile-removing-a-user-from-shared-folder-view-shared-with-tab-visually-removes-the-user-below-too
#### Testing instructions  
Add `export SHARED_FOLDERS_ENABLED='1'` to env.sh to enable shared folders.
Share a folder with multiple users.
Remove a user (from the top of the list).
The animation should be smooth, and the UI should only hide the user you removed.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
